scripts/core/editor3/components/embeds/EmbedBlock.tsx

```diff
@@ -59,14 +63,22 @@ export class EmbedBlockComponent extends React.Component<IProps> {
 
         tree.find('script').each((i, s) => {
             if (s.hasAttribute('src')) {
-                let url = s.getAttribute('src');
-                const async = s.hasAttribute('async');
+                const newScript = document.createElement('script');
+                const oldKeys = Object.keys(window);
 
-                if (url.startsWith('http')) {
-                    url = url.substring(url.indexOf(':') + 1);
-                }
+                newScript.setAttribute('src', s.getAttribute('src'));
+                newScript.setAttribute('async', s.hasAttribute('async') ? 'async' : '');
+                document.body.appendChild(newScript);
```

Comment:
Would be good remove `newScript` in case component unmounts before script loads. It could be done using `cleanupFunctionsToRunBeforeUnmounting` that are used elsewhere.